code/modules/virus2/disease2.dm

```diff
@@ -32,7 +33,7 @@ var/global/list/disease2_list = list()
 	e.chance = rand(1, e.max_chance)
 	return e
 
-/datum/disease2/disease/proc/makerandom(var/greater=0)
+/datum/disease2/disease/proc/makerandom(var/greater = FALSE, patient_zero)
```

Comment:
Is it okay to not have the patient zero variable be declared?

Reply:
It doesn't look like it's used

Reply:
Upon testing it doesn't work.